packages/zero-height/src/convertToCss.ts

```diff
@@ -0,0 +1,52 @@
+// 🐉 here be dragons 🐲
+const flattenObject = (obj: any, prefix = '') =>
+  Object.keys(obj).reduce((acc, k) => {
+    const pre = prefix.length ? prefix + '-' : ''
+    if (typeof obj[k] === 'object')
+      Object.assign(acc, flattenObject(obj[k], pre + k))
+    //@ts-ignore
+    else acc[pre + k] = obj[k]
+    return acc
+  }, {})
```

Comment:
Another way to resolve the ts ignore:
```suggestion
  }, {} as any)
```

Reply:
You could also use a more specific type like `Record<string, any>`, to give us a little useful typing.

I'd also pass the type as a generic argument in the reducer function instead of casting the initial value (But that's more personal preference I think)